testsuite/tests/basic/opt_variants.ml

```diff
@@ -1,7 +1,9 @@
-(* TEST *)
+(* TEST
+unset DOES_NOT_EXIST
```

Comment:
```suggestion
unset FOO
```

Reply:
I don't agree - the other instances set a variable where the purpose of using `unset` is to ensure that it was not set by the environment already. In 80a403, I changed the one other instance of an environment which is expected not to exist to be `DOES_NOT_EXIST` too. The other `FOO`, `FOO2`, etc. ones all get passed to `putenv` or similar, so the idea there is to ensure that the value they get really came from `putenv` and not randomly from the environment.

Reply:
Makes sense.